### ISSUE TITLE
How did you solve the elevation proxy problem?

### ISSUE THREAD

**Issue by nemesisdesign** [MrMufflon/Leaflet.Elevation#38]
Hi,

this is a wonderful leaflet plugin.

We are using it in [nodeshot](https://github.com/ninuxorg/nodeshot) but we needed to develop a proxy to the Google Elevation API in order to be able to display elevation profiles on request (the Google Elevation API does not include a CORS header, so it can only be used through a proxy).

After a while we decided it would be useful to separate the geojson elevation proxy to the Google Elevation API in a standalone project so we just released [python-geojson-elevation](https://github.com/ninuxorg/python-geojson-elevation), which is just at version 0.1 but we plan to improve it and add a few easy to use wrappers for popular python web frameworks like django and flask.

We are curious to know how other people solved the same problem.

**Comment by MrMufflon** [comment]
Hi, 
thank you. I really like the integration in your application! 
I don't know about other solutions than either proxying the service or utilize CORS. This plugin was developed with static, locally available data in mind, so using a n elevation service is the next level...

**Comment by nemesisdesign** [comment]
Hi @MrMufflon,

glad you like the idea. The library [python-geojson-elevation](https://github.com/ninuxorg/python-geojson-elevation) is still pretty basic, but we are planning to improve it and add more features.

These are the features we'd like to have:

* support more Elevation web services (eg: MapQuest)
* flask wrapper
* django wrapper
* command line tool to automatically launch a web server with the proxy, very useful for development

Since our tool it's specifically designed to work with **Leaflet.Elevation**, it could be useful to inform somehow the users of this library to know about the existence of python-geojson-elevation, so they might want to join forces to improve a common tool.

What do you think?

Federico

**Comment by MrMufflon** [comment]
Sounds great. What do you have in mind? A link on the readme?

**Comment by nemesisdesign** [comment]
Yes that would be good. We could add a section at the end of the README that explains the possibility of getting the elevation data from a web service and link the library.
